lua/entities/gmod_wire_expression2/core/init.lua

```diff
@@ -154,7 +154,8 @@ end
 
 function E2Lib.registerCallback(event, callback)
 	if not wire_expression_callbacks[event] then wire_expression_callbacks[event] = {} end
-	table.insert(wire_expression_callbacks[event], callback)
+	local currExt = E2Lib.currentextension
+	table.insert(wire_expression_callbacks[event], function(...) E2Lib.currentextension = currExt return callback(...) end)
```

Comment:
Can you make the function take and pass (a, b, c, d, e, f) instead of varargs?

I know this probably isn't going to be used in any performance heavy areas but varargs annoy me a bit since they don't jit.